openlibrary/catalog/marc/parse.py

```diff
@@ -279,11 +279,8 @@ def read_edition_name(rec: MarcBase) -> str:
     'end': 'eng',
     'enk': 'eng',
     'ent': 'eng',
-    'cro': 'chu',
     'jap': 'jpn',
     'fra': 'fre',
-    'gwr': 'ger',
```

Comment:
`sze` looks like it could be an ISO code for the https://en.wikipedia.org/wiki/Seze_language , which has nothing to do with `slo`, and I think `chu` -> `cro` has a similar ambiguity. Without comments, I'm not sure what that mapping is protecting against, but to me it looks like they are more likely to re-assign non-MARC language codes to unrelated languages. I imagine there were some historical records that those changes worked for, but this code can only protect against systematic and likely codes we might encounter through regular older catalog imports.

Reply:
I guess my natural bias is towards not changing things I don't understand, particularly since the code represents (or should) decades of accumulated knowledge, but I'd be hard pressed to argued for preserving such an ancient bit of cruft.